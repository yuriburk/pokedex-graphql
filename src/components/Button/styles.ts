import styled from 'styled-components';

export const Container = styled.button`
  background: #78c850;
  height: 54px;
  border-radius: 10px;
  border: 0;
  padding: 0 14px;
  width: 100%;
  color: black;
  font-weight: bold;
  margin-top: 16px;
  transition: background-color 0.2 ease 0.2;

  &:hover {
    background: '#78c850';
  }
`;
