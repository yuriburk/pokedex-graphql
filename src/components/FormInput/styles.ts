import styled, { css } from 'styled-components';

import Tooltip from '../Tooltip';

export const Container = styled.div`
  display: flex;
  flex-direction: column;
  flex: 1;
`;

export const Label = styled.label`
  font-weight: bold;
  margin-bottom: 12px;
`;

interface IInputContainerProps {
  isFocused: boolean;
  isFilled: boolean;
  isErrored: boolean;
}

export const InputContainer = styled.div<IInputContainerProps>`
  background: #232129;
  border-radius: 10px;
  padding: 14px;
  border: 2px solid #232129;
  color: ${(props) =>
    props.isFocused || props.isFilled ? '#ff9000' : '#666360'};
  ${(props) =>
    props.isErrored &&
    css`
      border-color: #c53030;
    `}
  ${(props) =>
    props.isFocused &&
    css`
      border-color: #ff9000;
    `}

  display: flex;
  align-items: center;

  & + div {
    margin-top: 12px;
  }

  input {
    flex: 1;
    background: transparent;
    border: 0;
    color: #f4ede8;
    width: 100%;

    &::placeholder {
      color: #666360;
    }

    &:-webkit-autofill,
    &:-webkit-autofill:hover,
    &:-webkit-autofill:focus,
    &:-webkit-autofill:active {
      -webkit-box-shadow: 0 0 0 50px #232129 inset;
      -webkit-text-fill-color: #f4ede8;
    }
  }

  svg {
    margin-right: 16px;
  }
`;

export const Error = styled(Tooltip)`
  height: 20px;
  margin-left: 16px;

  svg {
    margin: 0;
  }

  span {
    background: #c53030;
    color: #fff;

    &::before {
      border-color: #c53030 transparent;
    }
  }
`;
