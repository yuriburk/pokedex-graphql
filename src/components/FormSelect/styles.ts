import { CSSProperties } from 'react';
import styled from 'styled-components';

export const Container = styled.div`
  display: flex;
  flex-direction: column;
  flex: 1;

  margin-bottom: 8px;
`;

export const Label = styled.label`
  font-weight: bold;
  margin-bottom: 2px;
`;

export const customStyles = {
  container: (base: CSSProperties): CSSProperties => ({
    ...base,
    flex: 1,
  }),
  control: (base: CSSProperties): Record<string, unknown> => ({
    ...base,
    backgroundColor: '#232129',
    borderColor: '#232129',
    padding: '6px',
    boxShadow: 'none',
    borderRadius: '10px',
    '&:hover': {
      borderColor: '#ff9000',
    },
  }),
  multiValue: (base: CSSProperties): CSSProperties => ({
    ...base,
    backgroundColor: '#1d1c24',
  }),
  multiValueLabel: (base: CSSProperties): CSSProperties => ({
    ...base,
    color: '#fff',
  }),
  menuList: (base: CSSProperties): CSSProperties => ({
    ...base,
    color: '#fff',
    backgroundColor: '#232129',
  }),
  option: (base: CSSProperties): Record<string, unknown> => ({
    ...base,
    backgroundColor: '#232129',
    '&:hover': {
      backgroundColor: '#1d1c24',
    },
  }),
  singleValue: (base: CSSProperties): Record<string, unknown> => ({
    ...base,
    color: '#fff',
  }),
};
