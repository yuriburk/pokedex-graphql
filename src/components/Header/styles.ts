import styled from 'styled-components';

export const Container = styled.div`
  display: flex;
  align-items: center;
  justify-content: space-between;
  margin-bottom: 18px;

  svg {
    cursor: pointer;
  }
`;
