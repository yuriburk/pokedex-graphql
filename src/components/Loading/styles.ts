import styled from 'styled-components';

export const LoadingContainer = styled.div`
  position: fixed;
  top: 50%;
  left: 50%;
`;
