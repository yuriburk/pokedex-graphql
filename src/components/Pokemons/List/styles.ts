import styled, { css } from 'styled-components';
import { IPokemonTypes } from 'interfaces';

export const Container = styled.div``;

export const List = styled.ul`
  display: flex;
  justify-content: flex-start;
  flex-wrap: wrap;

  list-style: none;
`;

export const ListItem = styled.li`
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-between;
  flex: 1;

  cursor: pointer;
  min-height: 289px;
  padding: 14px 30px;
  border-radius: 8px;
  margin: 4px;
  background-color: ${(props) => props.theme.colors.secundaryDark};

  transition: transform ease 0.5s;

  &:hover {
    transform: translateY(-6px);
  }
`;

export const PokemonInfo = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
`;

export const Title = styled.h2`
  color: ${(props) => props.theme.colors.primaryWhite};
  margin-top: 8px;
  word-break: break-word;
  overflow: hidden;
  text-overflow: ellipsis;
  display: -webkit-box;
  -webkit-line-clamp: 2;
  -webkit-box-orient: vertical;
`;

export const TextInfo = styled.p`
  color: white;
`;

export const SpecialContainer = styled.div`
  display: flex;
  justify-content: center;
  flex-wrap: wrap;

  margin-bottom: 22px;
`;

interface ISpecialInfoContainerProps {
  pokemonType: IPokemonTypes;
}

export const SpecialInfoContainer = styled.div<ISpecialInfoContainerProps>`
  padding: 4px 8px;
  margin: 4px 4px 0 0;
  border-radius: 12px;
  ${(props) => props.theme.typesColors[props.pokemonType]};
`;

export const SpecialInfo = styled.p`
  color: ${(props) => props.theme.colors.primaryWhite};
`;

export const Image = styled.img`
  width: 124px;
  height: 124px;
  border-radius: 8px;
`;

export const ButtonContainer = styled.div`
  display: flex;
  justify-content: flex-end;
`;
