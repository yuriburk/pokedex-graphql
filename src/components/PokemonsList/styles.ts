import styled from 'styled-components';

export const Container = styled.ul`
  display: flex;
  flex-wrap: wrap;

  list-style: none;
`;

export const Item = styled.li`
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  flex: 1;

  padding: 15px;
  border-radius: 5px;
  margin: 5px;

  background-color: white;
`;

export const Image = styled.img`
  width: 125px;
  height: 125px;
`;

export const Title = styled.h2`
  color: black;

  margin-top: 15px;
`;
