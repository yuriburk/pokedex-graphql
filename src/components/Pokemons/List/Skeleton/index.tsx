import React, { useMemo } from 'react';

import { ListItem, List } from '../styles';
import { SkeletonThemeContainer, SkeletonItem } from './styles';

const PokemonSkeleton: React.FC = () => {
  const generateSkeletons = useMemo(
    () =>
      Array.from(Array(18).keys()).map((key) => (
        <ListItem key={key}>
          <SkeletonItem width={140} height={124} />
          <SkeletonItem height={15} width={80} />
          <SkeletonItem
            height={25}
            width={50}
            style={{ borderRadius: '14px', marginBottom: '24px' }}
          />
        </ListItem>
      )),
    [],
  );

  return (
    <SkeletonThemeContainer color="#202020" highlightColor="#444">
      <List>{generateSkeletons}</List>
    </SkeletonThemeContainer>
  );
};

export default PokemonSkeleton;
