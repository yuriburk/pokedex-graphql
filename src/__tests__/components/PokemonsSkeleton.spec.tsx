import React from 'react';
import { render, cleanup, fireEvent, waitFor } from '@testing-library/react';

import { ThemeProvider } from 'styled-components';
import { theme } from 'styles/theme';
import { HomeSkeleton } from 'components/Skeleton/Home';

const PokemonsSkeletonWrapper = () => (
  <ThemeProvider theme={theme}>
    <HomeSkeleton />
  </ThemeProvider>
);

describe('PokemonsSkeleton Component', () => {
  afterEach(cleanup);

  it('should be able to take a snapshot', () => {
    const { asFragment } = render(<PokemonsSkeletonWrapper />);

    expect(asFragment()).toMatchSnapshot();
  });

  it('should be able to render 18 list items', async () => {
    const { getAllByTestId } = render(<PokemonsSkeletonWrapper />);

    const items = getAllByTestId('list-item');

    expect(items).toHaveLength(18);
  });
});
