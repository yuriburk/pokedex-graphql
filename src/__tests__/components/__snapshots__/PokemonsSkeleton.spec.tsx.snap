// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonsSkeleton Component should be able to take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-1wwgopu-SkeletonTheme"
  >
    <div
      style="display: flex; justify-content: space-between; margin-bottom: 8px;"
    >
      <span>
        <span
          class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
          style="width: 190px; height: 40px;"
        >
          ‌
        </span>
      </span>
      <span>
        <span
          class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
          style="width: 190px; height: 40px;"
        >
          ‌
        </span>
      </span>
    </div>
    <div
      class="css-1wwgopu-SkeletonTheme"
    >
      <ul
        class="sc-AxhCb gzfbtb"
      >
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
        <li
          class="sc-AxhUy dOuvcL"
          data-testid="list-item"
        >
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 140px; height: 124px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="width: 80px; height: 15px;"
            >
              ‌
            </span>
          </span>
          <span>
            <span
              class="react-loading-skeleton sc-AxirZ mmKyu css-1vmnjpn-skeletonStyles-Skeleton"
              style="border-radius: 14px; margin-bottom: 24px; width: 50px; height: 25px;"
            >
              ‌
            </span>
          </span>
        </li>
      </ul>
    </div>
  </div>
</DocumentFragment>
`;
